geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/AlterOfflineDiskStoreCommand.java

```diff
@@ -101,41 +97,33 @@ public Result alterOfflineDiskStore(
               initialCapacityString, loadFactorString, compressorClassName, statisticsEnabledString,
               offHeapString, false);
 
-          result = ResultBuilder.createInfoResult(resultMessage);
+          return ResultModel.createInfo(resultMessage);
         } else {
-          result = ResultBuilder.createParsingErrorResult(
+          return ResultModel.createCommandProcessingError(
               "Cannot use the --remove=true parameter with any other parameters");
         }
       } else {
         if (remove) {
           DiskStoreImpl.destroyRegion(diskStoreName, dirs, "/" + regionName);
-          result = ResultBuilder.createInfoResult("The region " + regionName
+          return ResultModel.createInfo("The region " + regionName
               + " was successfully removed from the disk store " + diskStoreName);
         } else {
           // Please provide an option
-          result = ResultBuilder.createParsingErrorResult("Please provide a relevant parameter");
+          return ResultModel.createInfo("Please provide a relevant parameter");
         }
       }
       // Catch the IllegalArgumentException thrown by the modifyDiskStore function and sent the
     } catch (IllegalArgumentException e) {
-      String message = "Please check the parameters";
-      message += "\n" + e.getMessage();
-      result = ResultBuilder.createGemFireErrorResult(message);
+      return ResultModel.createInfo("Please check the parameters. " + e.getMessage());
```

Comment:
There should be unit tests for the exceptions thrown by the command. If tests had previously existed, they would have shown that this is a change in behavior, i.e. from and error result to an info result.